db/schema.rb

```diff
@@ -64,6 +64,7 @@
     t.string "subtitle"
     t.integer "parent_id"
     t.integer "position"
+    t.string "published_status"
     t.index ["position", "parent_id"], name: "index_content_pages_on_position_and_parent_id", unique: true
```

Comment:
I removed published_status,  lets not add it back until the story for status arrives

Reply:
if i remove that ill do it with a migration